doc/router_certificates.md

```diff
@@ -0,0 +1,31 @@
+# Router certificates
+
+The default KubeCF installation generates a `router_ca` and a `router_ssl` variables. The former
```

Comment:
```suggestion
The default KubeCF installation generates the `router_ca` and `router_ssl` variables. The former
```
It sounds really odd to have "a… and a… (thing)s", even though the numbers add up to be correct, IMHO.